plugin/batch-queue/job_examples/stampede/slurm_rstr.job

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+# Put your SLURM options here
+#SBATCH --time=00:30:00         # put proper time of reservation here
+#SBATCH --nodes=129             # number of nodes, where one node is the coordinator node, and the rest are the computing nodes
+#SBATCH --ntasks-per-node=16    # processes per node
+#SBATCH --job-name="restart"
+#SBATCH --output=restart.std
+#SBATCH --error=restart.err
+#SBATCH --partition=normal
+#SBATCH -A project_number
+
+# Start dmtcp coordinator on launching node. Free TCP port is automatically allocated.
+# this function creates dmtcp_command.$JOBID script that serves like a wrapper around
+# dmtcp_command that tunes it on exact dmtcp_coordinator (it's hostname and port)
```

Comment:
tunes it on -> tunes it for the
it's hostname -> its hostname
and port) -> and port).